bmi_calculator/bmi_calculator.py

```diff
@@ -0,0 +1,129 @@
+import sys
+
+from PyQt6.QtWidgets import (
+    QApplication,
+    QDialog,
+    QWidget,
+    QLabel,
+    QRadioButton,
+    QHBoxLayout,
+    QComboBox,
+    QButtonGroup,
+    QDialogButtonBox,
+    QFormLayout,
+    QLineEdit,
+    QVBoxLayout,
+    QPushButton,
+)
+class MyGUIWindow(QDialog):
```

Comment:
The class names should be meaning full   use` BMICalWindow(QDialog`) instead of MyGUIWindow(QDialog):